src/geo/points.ts

```diff
@@ -72,7 +79,34 @@ const distance = (pointFrom: Point, pointTo: Point): number | null => {
   return earthRadius * c
 }
 
+const equals = (point1: Point, point2: Point): boolean => {
+  if (point1 === null && point2 === null) return true
```

Comment:
can't this be if(point1===point2) return true ?

Reply:
ok, and null checks can be removed